controllers/home/views/home.njk

```diff
@@ -7,7 +7,7 @@
 {% set title = copy.title %}
 
 {% block content %}
-    <main role="main">
+    <main role="main" id="content">
```

Comment:
Looks like the Cypress tests are failing on the home page because we have two `content` IDs. If you search further down this page you should find there's another ID on a later element, we can probably remove that one if favour of this new one for consistency.

Reply:
Fixed, thanks!